### PR TITLE
Vanilla upgrade 2.0.1

### PR DESCRIPTION
**Pull request by kwm14**
## Done
- An updated version of the site to Vanilla 2.0.1
- Updated components, classes, and utilities based on release notes [here](https://github.com/canonical-web-and-design/vanilla-framework/releases/tag/v2.0.1)
- Visual update 😉 

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: [http://0.0.0.0:8005/](http://0.0.0.0:8005/)
4. Or view via demo link generated below 👍 

## Screenshot
conjure up io

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,166 +2,167 @@
...
   </div>
-  <div class="row u-equal-height p-divider">
+  <div class="row p-divider">
     <div class="col-4 p-divider__block">
-      <div class="p-heading-icon">
+      <div class="p-heading-icon--small">
         <div class="p-heading-icon__header">
-          <img class="p-heading-icon__img" src="{{ site.asset_path }}3fd62570-docs.svg" alt="" />
-          <h3 class="p-heading-icon__title">User manual</h3>
+          <img class="p-heading-icon__img" src="{{ site.asset_path }}3fd62570-docs.svg">
```
Comment (anthonydillon) [inline index.html]: This image has no `alt` tag.

File: index.html [hunk 2]
```diff
@@ -2,166 +2,167 @@
...
       </div>
     </div>
-
     <div class="col-4 p-divider__block">
-      <div class="p-heading-icon">
+      <div class="p-heading-icon--small">
         <div class="p-heading-icon__header">
-          <img class="p-heading-icon__img" src="{{ site.asset_path }}5c0fb313-ask+ubuntu.svg" alt="" />
-          <h3 class="p-heading-icon__title">Ask Ubuntu</h3>
+          <img class="p-heading-icon__img" src="{{ site.asset_path }}5c0fb313-ask+ubuntu.svg">
```
Comment (anthonydillon) [inline index.html]: Same here

File: index.html [hunk 3]
```diff
@@ -2,166 +2,167 @@
...
       </div>
     </div>
-
     <div class="col-4 p-divider__block">
-      <div class="p-heading-icon">
+      <div class="p-heading-icon--small">
         <div class="p-heading-icon__header">
-          <img class="p-heading-icon__img" width="100" src="{{ site.asset_path }}d3ae9c8e-irc-icon-circle.svg" alt="" />
-          <h3 class="p-heading-icon__title"><abbr title="Internet Relay Chat">IRC</abbr></h3>
+          <img class="p-heading-icon__img" src="{{ site.asset_path }}d3ae9c8e-irc-icon-circle.svg">
```
Comment (anthonydillon) [inline index.html]: Same here

**Comment by kwm14** [comment]
@anthonydillon nice catch. Updated and ready for re-review 👍

**Comment by anthonydillon** [comment]
LGTM :+1:

**Pull request merged**